### DISCUSSION TITLE
Window swallowing feature

### DISCUSSION THREAD

**Discussion by Malex** [qtile/qtile#3815] [Ideas] [answered]
Hello,

I've recently discovered this functionality in xmonad (which I very rarely use), called Window Swallowing: https://hackage.haskell.org/package/xmonad-contrib-0.17.0/docs/XMonad-Hooks-WindowSwallowing.html

This feature, while not always perfect by any means, is very cool and useful for anybody used (or that has to) run gui applications from terminals.

Is there any chace to get this on qtile, or is it for some design reason not possible/not desired?

I'm not good at haskell so I cannot really find where they implement this to see how they handled it

**Comment by elParaguayo** [comment, marked as answer]
Have a look at this thread (#1771) I think a solution was proposed in there.

**Comment by Malex** [comment]
search for whatever reason didnt point me to that. I'll take a look
